### DISCUSSION TITLE
QM9 dataset- different versions?

### DISCUSSION THREAD

**Discussion by arunraja-hub** [pyg-team/pytorch_geometric#9157] [Q&A]
Hi @rusty1s , I am using the QM9 dataset on PyG. The labels available in the PyG dataset are as follows: `'mol_id,A,B,C,mu,alpha,homo,lumo,gap,r2,zpve,u0,u298,h298,g298,cv,u0_atom,u298_atom,h298_atom,g298_atom'`

However, according to the [QM9 paper](https://www.nature.com/articles/sdata201422), the labels are as follows:
![image](https://github.com/pyg-team/pytorch_geometric/assets/43485111/51678050-20dd-4a3f-8f8f-1bbf302dacad)
I am looking for Mulliken partial charges in particular which cannot be found in the PyG version. Are there different versions or sources of the QM9 dataset files and is PyG using a particular one?

Thanks!

**Comment by rusty1s** [comment]
I created the dataset with the 13 targets from the [Gilmer](https://arxiv.org/pdf/1704.01212.pdf) paper in mind. I haven't checked if Mulliken partial charges are also available as targets. If they are, there shouldn't be a problem to expose them as well.
